.circleci/config.yml

```diff
@@ -421,10 +418,6 @@ workflows:
           requires:
             - hold
   release-candidate:
```

Comment:
Do you want to add an entirely separate job maybe called "manual-qe-workflow" or something? Where it does not depend on a new tag and we an trigger it manually with that `qe-workflow` param that we had?